ESPixelStick/src/platformDefinitions/GPIO_Defs_ESP32_Olimex_Gateway.hpp

```diff
@@ -0,0 +1,106 @@
+#pragma once
+/*
+* GPIO_Defs_ESP32_Olimex_Gateway.hpp - Output Management class
+*
+* Project: ESPixelStick - An ESP8266 / ESP32 and E1.31 based pixel driver
+* Copyright (c) 2021 Shelby Merrick
+* http://www.forkineye.com
+*
+*  This program is provided free for you to use in any way that you wish,
+*  subject to the laws and regulations where you are using it.  Due diligence
+*  is strongly suggested before using this code.  Please give credit where due.
+*
+*  The Author makes no warranty of any kind, express or implied, with regard
+*  to this program or the documentation contained in this document.  The
+*  Author shall not be liable in any event for incidental or consequential
+*  damages in connection with, or arising out of, the furnishing, performance
+*  or use of these programs.
+*
+*/
+
+/*
+ * Pinout for Olimex ESP32-GATEWAY
+ * https://www.olimex.com/Products/IoT/ESP32/ESP32-GATEWAY/open-source-hardware
+ */
+#define SUPPORT_ETHERNET
+
+//Output Manager
+#define SUPPORT_UART_OUTPUT
+#define DEFAULT_UART_1_GPIO     gpio_num_t::GPIO_NUM_4     // Supposed to be SD Card, but R10 not populated
+#define UART_LAST               OutputChannelId_UART_1
+
+
+#define SUPPORT_RMT_OUTPUT
+#define DEFAULT_RMT_0_GPIO      gpio_num_t::GPIO_NUM_12    // Tested working
```

Comment:
We have removed the use of RMT channel 0 due to a conflict with the UDP processing (Root cause unknown at this time). Also, You should use as many UART channels as possible (in this case two) because they have far lower ISR processing loads.

Reply:
Moved GPIO12 to UART 2